File: C# Project/ConsoleApp1/BilgiPaketi.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ConsoleApp1
{
    using System;
using System.Collections.Generic;

public class BilgiPaketi
{
    private List<Ders> tumDersler;

    public BilgiPaketi()
    {
        tumDersler = new List<Ders>();
    }

    public void DersEkle(Ders ders)
    {
        tumDersler.Add(ders);
    }

    public void TranskriptSorgula(Ogrenci ogrenci)
    {
        foreach (Ders ders in tumDersler)
        {
            if (ders.OgrenciKayitliMi(ogrenci.getNumara()))
            {
                Console.WriteLine($"{ders.dersAdi} adlı dersin kredisi {ders.kredi}, harf notu {ders.HarfNotu(0)}");
            }
        }
    }
}

}

File: Python Project/BilgiPaketi.py
class BilgiPaketi:
    def _init_(self):
        self.tum_dersler = []

    def ders_ekle(self, ders):
        self.tum_dersler.append(ders)

    def transkript_sorgula(self, ogrenci):
        print(f"{ogrenci.get_ad_soyad()} Ders Listesi:")
        for ders in self.tum_dersler:
            if ders.ogrenci_kayitli_mi(ogrenci.numara):
                print(f"{ders.ders_adi} dersi")
                print(f"Dersin kredisi: {ders.kredi}")
                print(f"Harf notunuz: {ders.harf_notu(0)}")
